CHANGES.md

```diff
@@ -1,3 +1,12 @@
+# Changes in Version 5
+
+## Summary of breaking changes
+
+- PostgreSQL 9.5 is no longer supported by this role.
+- `postgresql_install_server` is removed, the server is always configured, use `ome.postgresql_client` to install just the client.
```

Comment:
:+1:

Reply:
Is it worth failing if it's set to `False`?

Reply:
Done in https://github.com/ome/ansible-role-postgresql/pull/19/commits/73bb1101803f7129c4149bd55602ef686b0d3d06